runtime/service.go

```diff
@@ -52,6 +54,7 @@ import (
 const (
 	defaultVsockPort     = 10789
 	supportedMountFSType = "ext4"
+	namedNetNSPath       = "/var/run/netns/"
```

Comment:
Just curious, what's the reason we are requiring the ns come from under `/var/run/netns`? As far as I knew you can bind a net NS fd anywhere you want on a filesystem, is this just a standard location?

Reply:
Named network namespaces are typically created under `/var/run/netns`. See `ip-netns(8)` [man page](http://man7.org/linux/man-pages/man8/ip-netns.8.html) for details. I have not seen too many (any tbh) implementations that use an alternative path.  You're right that the netns fd can certainly be bound anywhere on the filesystem. However, because the assumption is the use of named network namespaces, this path was chosen. An alternative could be letting clients specify `FirecrackerNetworkNamespacePath` instead of `FirecrackerNetworkNamespace`. But, it might be a bit much to specify was my take on it.

Reply:
Thanks, that makes sense. The other middle-ground option I can think of is check if the user specified an absolute or relative path; if they specified an absolute path use that unchanged, if it was relative append it to the end of `/var/run/netns/`.

If you agree that makes sense it's probably worth updating, but I don't consider it a blocker given it's just to support an obscure use-case.

Reply:
Since the most widely used library I'm aware of (for dealing with network namespaces in golang) makes this same assumption: https://github.com/vishvananda/netns/blob/13995c7128ccc8e51e9a6bd2b551020a27180abd/netns_linux.go#L81, I'd prefer to keep it the way it is. Let me know if you feel strongly about it and we can make that change.

Reply:
I see, no that works for me for now, easy to update in the future if ever needed.

Reply:
> If fully specified, we should use the concrete path rather than a relative path. This is useful when the ns is not bind-mounted anywhere but is instead one of the ns files from `/proc`, like `/proc/<pid>/ns/net`.

Agree. I'll make this change since two of you have asked for it. But, I still don't believe that clients would want to do that. It's much easier to manage/deal with named network namespaces and that's what will drive most of the usage of this field imo.